source/numerics.tex

```diff
@@ -18436,6 +18478,166 @@
 \tcode{ret.first}.
 \end{itemdescr}
 
+\rSec3[simd.bit]{\tcode{basic_simd} bit library}
+
+\begin{itemdecl}
+template<@\exposconcept{simd-type}@ V> constexpr V byteswap(const V& v) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+The type \tcode{V::value_type} models \tcode{integral}.
+
+\pnum
+\returns
+A \tcode{basic_simd} object where the $i^\text{th}$ element is initialized to
+the result of \tcode{std::byteswap(v[$i$])} for all $i$ in the range
+\range{0}{V::size()}.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<@\exposconcept{simd-type}@ V> constexpr V bit_ceil(const V& v) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+The type \tcode{V::value_type} is an unsigned integer type
+\iref{basic.fundamental}.
```

Comment:
I think this needs to come flush after the preceding word, otherwise we won't
get the appropriate line-break penalties.